pandas-stubs/io/pytables.pyi

```diff
@@ -18,11 +22,6 @@ from pandas._typing import (
 
 from pandas.core.dtypes.generic import ABCExtensionArray
 
-# from tables import Col, File, Node
-# pytables may not be installed so create them as dummy classes
-class Col: ...
-class Node: ...
-
```

Comment:
disagree here.  If you install `pandas`, you don't have to install `pytables`, so this means if you install `pandas-stubs`, you have to install `pytables`.  I would leave this as is.

Reply:
It is only needed for the development mode but I agree it doesn't add much value.